data/templates/nginx/plain/yunohost_admin.conf

```diff
@@ -13,23 +13,16 @@ server {
 
 server {
     # Disabling http2 for now as it's causing weird issues with curl
-    #listen 443 ssl http2 default_server;
-    #listen [::]:443 ssl http2 default_server;
-    listen 443 ssl default_server;
-    listen [::]:443 ssl default_server;
+    listen 443 ssl http2 default_server;
+    listen [::]:443 ssl http2 default_server;
 
     ssl_certificate /etc/yunohost/certs/yunohost.org/crt.pem;
     ssl_certificate_key /etc/yunohost/certs/yunohost.org/key.pem;
     ssl_session_timeout 5m;
     ssl_session_cache shared:SSL:50m;
 
     # As suggested by Mozilla : https://wiki.mozilla.org/Security/Server_Side_TLS and https://en.wikipedia.org/wiki/Curve25519
-    # (this doesn't work on jessie though ...?)
-    # ssl_ecdh_curve secp521r1:secp384r1:prime256v1;
-
-    # As suggested by https://cipherli.st/
-    ssl_ecdh_curve secp384r1;
-
+    ssl_ecdh_curve X25519:sect571r1:secp521r1:secp384r1:prime256v1;
```

Comment:
This is not the recommendation from Mozilla ... Where is the source explaining why we should have X25519 and sect571r1

Reply:
Anyway : if you see on some source : these 2 curves are not supported by all browser, and the changes may need to be applied in openssl conf). I kept the 2 other ecdh for backward compatibility. For X25519, it's in the wiki page (link is already in comment of the PR) : 

For source : 
- https://github.com/h5bp/server-configs-nginx 
- https://github.com/stylersnico/nginx-secure-config
These are conf which have the goal to be updated following the latest security features.

Reply:
ok. So, I close the PR, and never will touch the nginx conf anymore (we have already good conf). I don't even understand why I'm spending hours in doing research for nothing.

Just one point : here I don't understand what is the problem as there is backward compatibility already, and if you see X25519 wikipedia, you would have plenty of source of researcher from several university which will tell you that it has quite good feature (robust, quick...). :confused: 

I trust several other conf too... Being blocked by Mozilla configuration, is a bit strange, as the mozilla conf is not the best at all today.